META.d/_summary.yaml

```diff
@@ -0,0 +1,13 @@
+schema: 1.1
+
+partition: internal-platform
+category: github-action
+
+summary:
+  name: actions-docker-build
+  owner: team-rel-eng
+  pagerduty: https://hashicorp.pagerduty.com/service-directory/P37IMEL
```

Comment:
I think this isn't present anymore in 1.1, and you could include it in links.yml

Reply:
Thanks, Martin! I appreciate it